formats/service_manual_guide/frontend/examples/point_page.json

```diff
@@ -17,6 +17,23 @@
         "title": "Why it's in the standard",
         "href": "#why-its-in-the-standard"
       }
+    ],
+    "change_history": [
+      {
+        "public_timestamp": "2015-10-07T08:17:10+00:00",
+        "note": "This is our latest change",
+        "reason_for_change": "This is the reason for our latest change"
+      },
+      {
+        "public_timestamp": "2015-09-09T08:17:10+00:00",
+        "note": "This is another change",
+        "reason_for_change": "This is why we made this change\nand it has a second line of text"
+      },
+      {
+        "public_timestamp": "2015-09-01T08:17:10+00:00",
+        "note": "Guidance created",
```

Comment:
This could be "Guidance first published" to be a little closer to real life. Until it changes of course..